src/style.css

```diff
@@ -0,0 +1,92 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+.big {
+  position: relative;
+  margin-top: 20px;
+}
+
+.big-field {
+  padding: 0 62px;
+  width: 100%;
+  height: 70px;
+  font-family: Avenir, Arial, sans-serif;
+  font-weight: 300;
+  font-size: 16px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.big-icon {
+  position: absolute;
+  left: 26px;
+  top: 26px;
+}
+
+.small {
+  position: relative;
+  margin-top: 20px;
+}
+
+.small-field {
+  padding: 0 33px;
+  width: 100%;
+  height: 42px;
+  font-family: Avenir, Arial, sans-serif;
+  font-weight: 300;
+  font-size: 14px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.small-icon {
+  height: 11px;
+  position: absolute;
+  left: 13px;
+  bottom: 16px;
+}
+
+.big-field:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.big-field:focus {
+  font-family: Avenir, Arial, sans-serif ;
+  font-weight: 800;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+}
+
+.small-field:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.small-field:focus {
+  font-family: Avenir, Arial, sans-serif;
+  font-weight: 800;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+}
```

Comment:
The same case here with duplicating. [Checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md), the 2nd point.